packages/RichTextEditing-Core.package/RichTextEditor.class/instance/makeTextURLFrom..st

```diff
@@ -0,0 +1,6 @@
+pasting
+makeTextURLFrom: aString
+
+	^ aString asText
+			addAttribute: (TextURL new url: aString);
+			addAttribute: (TextEmphasis underlined)
```

Comment:
maybe make it blue?

Reply:
TextURL automatically colors the text blue.